composer.json

```diff
@@ -5,8 +5,9 @@
   "homepage": "http://drupal.org/project/graphql",
   "license": "GPL-2.0+",
   "require": {
-    "php": ">=7.3",
-    "webonyx/graphql-php": "^14.8.0"
+    "drupal/typed_data": "^1.0 || ^2.0",
+    "php": ">=8.0",
+    "webonyx/graphql-php": "^15.12.0"
```

Comment:
do we need to bump the graphql-php dependency?

Reply:
I guess yes the last tag of 14.11.0 supports bare minimum package with php minimum support of 7.1 and phpunit maximum support is ^8.5
whereas latest tag for ^15.12.5 minimum php support is 7.4 and phpunit 9.5 and 10, also added few new packages under require-dev section.
I suggest we should bump this package version to make things work with D10 and above.